libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -277,12 +278,6 @@ export class StripeClient {
     return result as StripeResponse<StripeInvoice>;
   }
 
-  @Cacheable({
```

Comment:
Payment intent values, e.g. status, can change during a checkout session. Therefore the retrieval should not be cached.